<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDefineControl</class>
 <widget class="QDialog" name="DlgDefineControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control surface definition</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabContainer">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFlap">
      <attribute name="title">
       <string>Flap</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout2">
         <item row="0" column="3">
          <widget class="QPushButton" name="pbNewFlap">
           <property name="text">
            <string>&amp;New flap</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Control surface</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="cbSelectWing"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QRadioButton" name="rbTrailingEdge">
           <property name="text">
            <string>Trailing edge flap</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Attach to wing</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="pbDeleteFlap">
           <property name="text">
            <string>&amp;Delete flap</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="cbFlapName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtCurrent</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="rbLeadingEdge">
           <property name="text">
            <string>Leading edge flap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pbMirrorCopy">
           <property name="text">
            <string>Mirror &amp;copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Edit hinge point</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="sbEditHinge">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pbNewHinge">
           <property name="text">
            <string>New &amp;hinge</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Spanwise position</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="sbSpanwisePos">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.200000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Chordwise position</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="sbChordwisePos">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>20</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.700000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPattern">
      <attribute name="title">
       <string>Pattern</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="hboxLayout2">
         <item>
          <widget class="QComboBox" name="cbPattern">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbNewPattern">
           <property name="text">
            <string>&amp;New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDeletePattern">
           <property name="text">
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tablePattern">
         <column>
          <property name="text">
           <string>Flap segment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Participation</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hboxLayout1">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabContainer</tabstop>
  <tabstop>cbFlapName</tabstop>
  <tabstop>cbSelectWing</tabstop>
  <tabstop>rbTrailingEdge</tabstop>
  <tabstop>rbLeadingEdge</tabstop>
  <tabstop>pbNewFlap</tabstop>
  <tabstop>pbMirrorCopy</tabstop>
  <tabstop>pbDeleteFlap</tabstop>
  <tabstop>sbEditHinge</tabstop>
  <tabstop>pbNewHinge</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>cbPattern</tabstop>
  <tabstop>pbNewPattern</tabstop>
  <tabstop>pbDeletePattern</tabstop>
  <tabstop>tablePattern</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbOk</sender>
   <signal>released()</signal>
   <receiver>DlgDefineControl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>released()</signal>
   <receiver>DlgDefineControl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
